apps/crc_sus.py

```diff
@@ -4,28 +4,24 @@
 and will not work without a running bank installation.
 """
 
-import grp
-import os
 from argparse import Namespace
 from typing import Dict
 
-import dataset
+from bank.account_logic import AccountServices
 
 from .utils.cli import BaseParser
-from .utils.system_info import Slurm
+from .utils.system_info import Shell
 
 
 class CrcSus(BaseParser):
     """Display the number of service units allocated to an account."""
 
-    banking_db_path = 'sqlite:////ihome/crc/bank/crc_bank.db'
-
     def __init__(self) -> None:
         """Define the application commandline interface"""
 
         super().__init__()
-        default_group = grp.getgrgid(os.getgid()).gr_name
-        help_text = "slurm account name (defaults to the current user's primary group name)"
+        default_group = Shell.run_command("id -gn")
```

Comment:
Is using `Shell` or `grp` the preferred method for this? I can switch crc-proposal-end to us `grp` if that's how we want to do it.

Reply:
Generally running things via the shell is discouraged due to security reasons. In some cases it's unavoidable (like for Slurm).